spec/factory_bot/definition_proxy_spec.rb

```diff
@@ -56,7 +56,7 @@
     expect(subject).to have_dynamic_declaration(:attribute_name).with_value(attribute_value)
   end
 
-  it "calls super" do
+  it "raises a NoMethodError" do
     invalid_call = -> { proxy.static_attributes_are_gone true }
     expect(invalid_call).to raise_error(NoMethodError)
```

Comment:
Since the benefit of doing this is for readability, what are your thoughts on expanding the `raise_error` expectation to also verify we inform the developer of what's happening?

Reply:
Yup, that makes sense